e2e/tests/baseimages_update.yaml

```diff
@@ -0,0 +1,7 @@
+kind: BaseImage
+name: nodejs-base
+imageUrl: dispatchframework/nodejs-base:0.0.8
+language: python3
```

Comment:
This is weird. We are grabbing a nodejs base image but saying the language is python3. I think you just cut/pasted this from the images_update.yaml. Not sure why it was like this in that file...

Reply:
Yeah, that's what the test looked like originally, so I just left it.  But I too thought it odd.

Reply:
Sooo.... can we fix it in this PR? :-)